modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc

```diff
@@ -27,12 +27,16 @@ Only public and AWS PrivateLink clusters are supported with STS. Regular private
 +
 [IMPORTANT]
 ====
-The EBS operator role is required in addition to the account roles to successfully create your cluster.
+The EBS Operator role is required in addition to the account roles to successfully create your cluster.
+
+This role must be attached with the `ManagedOpenShift-openshift-cluster-csi-drivers-ebs-cloud-credentials` policy, an IAM policy required by ROSA to manage back-end storage through the Container Storage Interface (CSI).
+
+For more information about the policies and permissions that the cluster Operators require, see xref:../rosa_architecture/rosa-sts-about-iam-resources.adoc#rosa-sts-account-wide-roles-and-policies-creation-methods_rosa-sts-about-iam-resources[Methods of account-wide role creation].
```

Comment:
We can't use `xrefs` in modules. Please refer to the OpenShift docs [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references).

If you want to call out a link for customers to follow in a module, you refer to the title of the link in quotes, then add [a trailing include additional resources section](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-assemblies) with the `xref` in the assembly.

If you run into to trouble, members of your team can help you out. Or you can ask for help in the team Slack channel.

Reply:
Thanks @michaelryanpeter, I will fix these.

Reply:
Oh man, sorry for missing this in my peer review @michaelryanpeter and @AedinC .

Reply:
I can't believe I forgot about it too @skrthomas!

Reply:
Not a problem, at all! I forget sometimes. :)

My kingdom for an xref...